<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPlainTextEdit, QLineEdit
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#c5e1a5;
  border:2px solid #9ccc65;
  border-radius:5px;
  color: #454545;
  padding: 6px 6px
}

/* labels and checkboxes */
QLabel,QCheckBox
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
}


/* 绿色按钮 */
QPushButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#9ccc65;
  border:2px solid #7cb342;
  border-radius:5px;
  color: white;
  padding: 6px 6px
}
QPushButton:hover
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#7cb342;
  border:2px solid #689f38;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover:pressed
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#558b2f; 
  border:2px solid #33691e;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="status_text">
     <property name="text">
      <string>信息</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>截取坐标：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="coords">
           <property name="text">
            <string>(0, 0)(0, 0)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btn_set_pos">
           <property name="styleSheet">
            <string notr="true">/* buttons */
QPushButton
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:rgb(248,187,208);
  border:2px solid #f06292;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color: #f48fb1;
  border:2px solid #ec407a;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}
QPushButton:hover:pressed
{
  font: 14pt &quot;Microsoft YaHei UI&quot;;
  background-color:#f44336; 
  border:2px solid #d81b60;
  border-radius:5px;
  color:white;
  padding: 6px 6px
}</string>
           </property>
           <property name="text">
            <string>指定坐标</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ocr">
           <property name="text">
            <string>截图+识别</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_search">
           <property name="text">
            <string>截图+搜索</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
